cli/src/main.rs

```
@@ -127,6 +131,23 @@ async fn main() {
                 }
                 None => {
                     let local_context = analyze_local_context().await.ok();
+                    let api_config: ClientConfig = config.clone().into();
+                    match get_or_build_local_code_index(&api_config, None).await {
+                        Ok(_) => {
```

Do we want to exit if the indexing fails?

---

if we fail silently here, all other tools will still be available, if we exit the agent will not be available as a whole if indexing has a bug or something